clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/actions/LoadFront50App.java

```diff
@@ -124,11 +124,11 @@ public Result apply(@NotNull LoadFront50AppCommand command, @NotNull Saga saga)
             "Failed to convert front50 application to internal model", e);
       }
     } catch (Exception e) {
-      log.error("Failed to load front50 application attributes for {}", command.getAppName(), e);
       if (command.isAllowMissing()) {
         // It's ok to not load the front50 application
         return new Result(command.nextCommand, Collections.emptyList());
       }
+      log.error("Failed to load front50 application attributes for {}", command.getAppName(), e);
```

Comment:
This little guy snuck in here on accident, but I'm not gonna make a PR just for this change anyway. This reduces false error log noise.